files/__addonLocation__/package.json

```diff
@@ -25,6 +25,9 @@
     "test": "echo 'A v2 addon does not have tests, run tests in test-app'",
     "prepack": "rollup --config"
   },
+  "peerDependencies": {
+    "ember-source": "^5.0.0"
```

Comment:
Presumably addons built this way could still target and test against older versions of `ember-source`.

In that case wouldn't they need to declare that range here, also?

Reply:
Yup

Reply:
4.12 is the latest v4 🙃 i confused myself